contracts/PledgeAgent.template

```diff
@@ -206,7 +206,7 @@ contract PledgeAgent is IPledgeAgent, System, IParamSubscriber {
     scores = new uint256[](candidateSize);
     for (uint256 i = 0; i < candidateSize; ++i) {
       Agent storage a = agentsMap[candidates[i]];
-      scores[i] = a.power * totalCoin * powerFactor / 10000 + a.coin * totalPower;
+      scores[i] = a.power * totalCoin * powerFactor / 10000 + a.coin * totalPower; //@openissue
```

Comment:
this does not match the documented formula for hybrid scores calculation here:
https://docs.coredao.org/core-white-paper-v1.0.5/satoshi-plus-consensus/validator-election

Reply:
my initial understanding was that the doc-to-code mapping is:

rHp = a.power (hash power delegated to validator)
tHp = totalPower  (total hash power on Core)
rSp = a.coin  (stake delegated to validator)
tSp = totalCoin  (total stake on Core)
m = powerFactor/10000  (dynamic weighting that adjusts over time..)

for these values the code calc (float or int) does not match the documented hybrid score calculation.
since this calc. is major to our system functioning as expected, I suggest we will take the time to figure this one out.

Reply:
Let's investigate and align on this on our call